src/panel.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <title>Block-facebook</title>
+      <link rel="stylesheet" href="panel.css">
+    </head>
+
+    <body>
+      <div id="panel-1" class="wrapper">
```

Comment:
As mentioned split this into two files and set the html page in the browser.js file.